pkg/config/template/template.go

```diff
@@ -0,0 +1,14 @@
+package template
+
```

Comment:
move the  strcuture  to core/controller/cluster/controller_operation.go

Reply:
All configurations of coreConfig are defined in pkg/config